src/vdaf/dummy.rs

```diff
@@ -123,6 +123,7 @@ impl vdaf::Aggregator<0, 16> for Vdaf {
     fn prepare_init(
         &self,
         _verify_key: &[u8; 0],
+        _ctx: &[u8],
```

Comment:
I think we should just apply `#[allow(clippy::too_many_arguments)]` to this method and the other one with a lint. This is just the most complicated part of the protocol, and I don't think encapsulating subsets of these parameters would be a net benefit.

In this case, we have the following:

- VDAF instance: what we're doing
- Verification key: shared secret
- Application context: what we're doing, more specifically
- Aggregator ID: who am I
- Aggregation parameter: what the Collector said
- Nonce: Identity of the Client's message
- Public share: What the Client said to everybody
- Input share: What the Client said to us

Maybe we could throw the last three references in a struct, but I know that callers would just construct that immediately before calling `prepare_init()`, and then never use it again, so it would be a false simplification. Perhaps we could push more into `Self`, but then we would have to start caring about how `Self` is reused.

I think we should just ignore this instance and call it good. FWIW, we have a couple precedents for this in the IDPF and Poplar1 modules.

Reply:
Ok, this makes sense. Will do